seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java

```diff
@@ -169,7 +169,9 @@ private int getBytesForValue(Object v, SeaTunnelDataType<?> dataType) {
                 if (elementType instanceof DecimalType) {
                     return ((Object[]) v).length * 36;
                 }
-
+                if (elementType instanceof MapType) {
+                    return getArrayMapNotNullSize(v);
+                }
```

Comment:
why not extend https://github.com/apache/seatunnel/blob/875fcd89c59faa4c7b97f3e2c1bf3be707a205cd/seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelRow.java#L213 to support parse map type? After this we can reuse `getBytesForValue` to parse map.